extensions/sql-database-projects/package.json

```diff
@@ -382,18 +396,22 @@
     }
   },
   "dependencies": {
-    "@types/xml-formatter": "^1.1.0",
     "@microsoft/ads-extension-telemetry": "^1.1.5",
+    "@types/which": "^2.0.1",
+    "@types/xml-formatter": "^1.1.0",
     "fast-glob": "^3.1.0",
     "fs-extra": "^5.0.0",
     "jsonc-parser": "^2.3.1",
     "promisify-child-process": "^3.1.1",
+    "semver": "^7.3.5",
     "vscode-languageclient": "^5.3.0-next.1",
     "vscode-nls": "^4.1.2",
+    "which": "^2.0.2",
     "xml-formatter": "^2.1.0",
     "xmldom": "^0.3.0"
   },
   "devDependencies": {
+    "@types/fs-extra": "^5.0.0",
```

Comment:
Dunno, but almost all of these (including `@types/fs-extra`) are alphabetical sorting of existing dependencies; I think `yarn` did that automatically.

The new ones are: `semver`, `which`, and `@types/which`.

Reply:
@kisantia to answer your question - the `@types` packages are for providing the typings files (.d.ts) for packages that don't bundle them in the package itself. This is pretty common for native JS packages since those don't have typings to begin with.

The `@types` packages will generally only contain the .d.ts file and nothing else - the "normal" package is the one that contains the actual functionality via *.js files.